cli/coverage_test.go

```diff
@@ -1,9 +1,31 @@
 package cli
 
 import (
+	"io/ioutil"
 	"testing"
 )
 
+func TestStripComments(t *testing.T) {
+
+	source, _ := ioutil.ReadFile("testdata/strip-comments/commented-source.txt")
+	target, _ := ioutil.ReadFile("testdata/strip-comments/expected-source.txt")
```

Comment:
I'd prefer to leave large chunks of text in a separate file, otherwise code becomes unreadable because there would be a mix of Go code and string data that looks quite similar to Go code.
My understanding is that `testdata` is a convention where test resources should be kept, content of this directory is ignored by `govendor` and `godep`